src/main/scala/net/snowflake/spark/snowflake/Utils.scala

```diff
@@ -71,8 +71,11 @@ object Utils {
   /**
     * Client info related variables
     */
-  private[snowflake] lazy val sparkAppName =
-    SparkContext.getOrCreate().getConf.get("spark.app.name", "")
+  private[snowflake] lazy val sparkAppName = if (SparkEnv.get != null) {
```

Comment:
To reviewers:
SparkContext is not available on executor. So, it is changed to use SparkEnv.